Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs

```diff
@@ -204,8 +204,8 @@ public void UpdateButtons()
             Respawn.Visible = _adminManager.CanCommand("respawn");
             Respawn.Disabled = !Respawn.Visible || disabled;
 
-            Teleport.Visible = _adminManager.CanCommand("tpto");
-            Teleport.Disabled = !Teleport.Visible || disabled;
+            Follow.Visible = _adminManager.CanCommand("tpto");
```

Comment:
shouldnt this be `CanCommand("follow")`

Reply:
yes it should be, I missed that bit when renaming it